cfme/tests/infrastructure/test_customization_template.py

```diff
@@ -1,101 +1,105 @@
 # -*- coding: utf-8 -*-
 import fauxfactory
 import pytest
-
-from cfme.infrastructure import pxe
-from cfme.utils import error
-from cfme.utils.blockers import BZ
+from cfme.utils import error, version
 from cfme.utils.update import update
 
 pytestmark = [pytest.mark.tier(3)]
 
 
-def test_customization_template_crud():
+@pytest.fixture(scope="module")
+def collection_init(appliance):
+    collection = appliance.collections.customization_template
+    return collection
```

Comment:
**REQUIRED**
```python
@pytest.fixture(scope="module")
def collection(appliance):
    return appliance.collections.customization_template
```
And below you can remove this redundant line: `collection = collection_init`